test/graphql/types/Organization/updater.test.ts

```
@@ -0,0 +1,260 @@
+	drizzleClient: {
+		query: {
+			usersTable: {
+				findFirst: ReturnType<typeof vi.fn>;
+			};
+			organizationsTable: {
+				findFirst: ReturnType<typeof vi.fn>;
+			};
+		};
+	} & GraphQLContext["drizzleClient"];
+
+	jwt: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)